From 1840c18cf837d712e4471b283de1a6bf46308fe2 Mon Sep 17 00:00:00 2001
From: Jan Petrous <jan.petrous@nxp.com>
Date: Fri, 24 Feb 2023 09:54:05 +0100
Subject: [PATCH 58/63] arm64: dts: s32g-pfe: remove unused global property
 nxp,pfeng-hif-channels

The property was redundant, the same HIF channel listing
can be obtained during DT scan on driver start-up.

Issue: AAVB-7057

Upstream-Status: Pending 

Signed-off-by: Jan Petrous <jan.petrous@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm64/boot/dts/freescale/s32g-pfe.dtsi | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/s32g-pfe.dtsi b/arch/arm64/boot/dts/freescale/s32g-pfe.dtsi
index 7583c657c938..a49da41be800 100644
--- a/arch/arm64/boot/dts/freescale/s32g-pfe.dtsi
+++ b/arch/arm64/boot/dts/freescale/s32g-pfe.dtsi
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: GPL-2.0
 /*
- * Copyright 2022 NXP
+ * Copyright 2022-2023 NXP
  *
  */
 
@@ -53,9 +53,6 @@ pfe: pfe@46000000 {
 			nxp,fw-util-name = "s32g_pfe_util.fw";
 			nvmem-cells = <&soc_revision>;
 			nvmem-cell-names = "soc_revision";
-			nxp,pfeng-hif-channels = <PFE_HIF_CHANNEL_0>,
-						 <PFE_HIF_CHANNEL_1>,
-						 <PFE_HIF_CHANNEL_2>;
 			nxp,pfeng-ihc-channel = <PFE_HIF_CHANNEL_0>;
 			status = "disabled";
 
-- 
2.25.1

